frontend/model/contracts/mailbox.js

```diff
@@ -98,63 +127,108 @@ sbp('chelonia/defineContract', {
         username: string,
         contractID: optional(string)
       }),
-      process ({ meta, data }, { state }) {
-        if (state.attributes.creator === data.username) {
-          if (state.users[meta.username]) {
-            throw new TypeError(L('Already existing direct message channel.'))
-          }
-        } else if (state.attributes.creator === meta.username) {
-          if (!state.users[data.username] || state.users[data.username].joinedDate) {
+      process ({ meta, data }, { state, getters }) {
+        const me = state.attributes.creator
```

Comment:
It's confusing to call this `me` - as it's not clear who "me" is referring to.

I recommend, to avoid confusion, writing this as something like:

```js
const { creator } = state.attributes
```

Reply:
Yeah, it's confusing actually. I think I should use `creator` instead and also add comment.

Reply:
I think I should add comment here. Actually, that is referring to `me` that's because `state.attributes.creator` is the creator who created this mailbox contract when registered the group income account.

Reply:
I would rename the variable to `creator` - keeping it as `creator`. "me", in English, is too confusing. These actions could potentially be called by anyone, in any context and any situation. And so for thinking clearly about this, `creator` makes more sense, because it's specific to what it's referring to. It doesn't even need a comment. But `me` doesn't make sense

Reply:
Okay, then. I will rename it to `creator` and add comment.